### PR TITLE
Code standard fixes

### PR DESCRIPTION
**Pull request by vedanshujain**
Applies code standard changes to wcpt plugin. 

Most of the changes are small, but these are some important ones:

- views/applications/meetup/shortcode-application.php
  - Added a nonce field `wcpt_event_application` to be used in application from.

- views/applications/wordcamp/shortcode-application.php
  - Added a nonce field `wcpt_event_application` to be used in application from.

- wcpt-event/class-event-admin.php
  - Added nonce check in `metabox_save`
  - Escaped output in `dislpay_meta_boxes`

- wcpt-event/class-event-application.php
  - Added nonce check in `render_application_shortcode`
  - Change definition of submit_application to pass nonce verified `$POST` arguments

- wcpt-loader.php
  - Indent whole file by 1 indent. ( Tip: To review, select a line and use browser search to find it. Its indented counterpart will be highlighted and can be used as an anchor to review code. If there is no match, then that line is changed, search again with a line above or below.)

- wcpt-meetup/class-meetup-admin.php
  - Added nonce check in `maybe_update_meetup_data`

- wcpt-wordcamp/wordcamp-admin.php
  - Escaping in `user_profile_wordcamp`, `column_data`
  - Escaping using kses in `post_row_actions`
  - Use post_data_raw instead of $_POST in `enforce_post_status`

### CODE REVIEW

File: public_html/wp-content/plugins/wcpt/views/applications/wordcamp/shortcode-application.php [hunk 1]
```diff
@@ -1477,8 +1477,9 @@ class="survey-email required" autocomplete="off" required/>
 
 			</div> <!-- PDF.pageInner -->
 		</div> <!-- PDF.pageOuter -->
+		<?php wp_nonce_field( 'wcpt_event_application', 'wcpt_event_application_nonce' ); ?>
```
Comment (vedanshujain) [inline public_html/wp-content/plugins/wcpt/views/applications/wordcamp/shortcode-application.php]: - [ ] Test if nonce field is getting rendered and submitted on application submit.

File: public_html/wp-content/plugins/wcpt/wcpt-event/class-event-admin.php [hunk 2]
```diff
@@ -402,15 +404,19 @@ public function display_post_states( $states ) {
...
+	 * @hook save_post
 	 */
 	public function metabox_save( $post_id, $post ) {
 		// Don't add/remove meta on revisions and auto-saves.
 		if ( wp_is_post_autosave( $post_id ) || wp_is_post_revision( $post_id ) ) {
 			return;
 		}
 
+		// Make sure the requset came from the edit post screen.
+		if ( empty( $_POST['_wpnonce'] ) || ! wp_verify_nonce( $_POST['_wpnonce'], 'update-post_' . $post_id ) ) {
```
Comment (vedanshujain) [inline public_html/wp-content/plugins/wcpt/wcpt-event/class-event-admin.php]: - [ ] Test if `metabox_save` is working, and not working if nonce is not verified.

File: public_html/wp-content/plugins/wcpt/wcpt-event/class-event-admin.php [hunk 3]
```diff
@@ -606,64 +609,65 @@ public static function display_meta_boxes(
 				<?php if ( 'checkbox' == $value ) : ?>
 
 					<p>
-						<strong><?php echo $key; ?></strong>:
-						<input type="checkbox" name="<?php echo $object_name; ?>"
-							   id="<?php echo $object_name; ?>" <?php checked( get_post_meta( $post_id, $key, true ) ); ?><?php echo $readonly; ?> />
+						<strong><?php echo esc_html( $key ); ?></strong>:
```
Comment (vedanshujain) [inline public_html/wp-content/plugins/wcpt/wcpt-event/class-event-admin.php]: - [ ] Test this and below fields with filled meta data and check if rendering properly.

File: public_html/wp-content/plugins/wcpt/wcpt-event/class-event-application.php [hunk 4]
```diff
@@ -61,8 +61,8 @@ public function enqueue_common_assets() {
 	public function render_application_shortcode() {
 		ob_start();
 
-		if ( isset( $_POST['submit-application'] ) ) {
-			$this->submit_application();
+		if ( wp_verify_nonce( $_POST['wcpt_event_application_nonce'], 'wcpt_event_application') && isset( $_POST['submit-application'] ) ) {
```
Comment (vedanshujain) [inline public_html/wp-content/plugins/wcpt/wcpt-event/class-event-application.php]: - [ ] Test if both WordCamp and Meetup applications can be submitted.

File: public_html/wp-content/plugins/wcpt/wcpt-event/class-event-application.php [hunk 5]
```diff
@@ -222,10 +224,11 @@ public function notify_applicant_application_received( $email_address, $meetup_c
 		//translators: Name and city of the event. Egs WordCamp in New York.
 		$message = sprintf(
 			__(
-				"Thank you for applying to organize a %s in %s! We'll send you a follow-up e-mail once we've had a chance to review your application.",
+				"Thank you for applying to organize a %1\$s in %2\$s! We'll send you a follow-up e-mail once we've had a chance to review your application.",
```
Comment (vedanshujain) [inline public_html/wp-content/plugins/wcpt/wcpt-event/class-event-application.php]: - [ ] Test if this message is rendering properly.

File: public_html/wp-content/plugins/wcpt/wcpt-event/class-event-application.php [hunk 6]
```diff
@@ -237,26 +240,27 @@ public function notify_applicant_application_received( $email_address, $meetup_c
...
 		$public_report_url = $this->get_application_report_url();
 		if ( isset( $public_report_url ) ) {
 			// `<%s|here> is syntax for slack message to hyperlink text `here` with url provided in `%s`
-			$message = sprintf( "%s. Public status can be followed <%s|here>.", $message, $public_report_url );
+			$message = sprintf( '%s. Public status can be followed <%s|here>.', $message, $public_report_url );
 		}
 
 		$default_status = $this->get_default_status();
-		$queue_size = wp_count_posts( $post_type=$this->get_event_type() )->$default_status;
+		$queue_size = wp_count_posts( $this->get_event_type() )->$default_status;
```
Comment (vedanshujain) [inline public_html/wp-content/plugins/wcpt/wcpt-event/class-event-application.php]: - [ ] Test if queue size number is correct.

File: public_html/wp-content/plugins/wcpt/wcpt-loader.php [hunk 7]
```diff
@@ -17,78 +18,78 @@
...
- * WCPT_Loader
- *
- * @package
- * @subpackage Loader
- * @since WordCamp Post Type (0.1)
- *
- */
-class WCPT_Loader {
-
+if ( ! class_exists( 'WCPT_Loader' ) ) :
```
Comment (vedanshujain) [inline public_html/wp-content/plugins/wcpt/wcpt-loader.php]: - [ ] Test if applications and admin view for Meetup and WordCamp loads properly.

File: public_html/wp-content/plugins/wcpt/wcpt-wordcamp/wordcamp-admin.php [hunk 8]
```diff
@@ -576,78 +581,78 @@ function column_headers( $columns ) {
...
 				}
 
-				if ( $wc_url = make_clickable( wcpt_get_wordcamp_url() ) ) {
+				$wc_url = make_clickable( wcpt_get_wordcamp_url() );
+				if ( $wc_url ) {
 					$wc['url'] = $wc_url;
 				}
 
-				echo implode( ' - ', (array) $wc );
+				echo wp_kses( implode( ' - ', (array) $wc ), wp_kses_allowed_html() );
```
Comment (vedanshujain) [inline public_html/wp-content/plugins/wcpt/wcpt-wordcamp/wordcamp-admin.php]: - [ ] Test if row actions are working fine.

**Comment by coreymckrill** [comment]
One potential pitfall with adding nonces to frontend forms is that the values will probably get cached by WP Super Cache, which could cause form submission problems. Off the top of my head, I'm not sure what the best workaround is.

**Comment by iandunn** [comment]
The only thing I'd change is that we typically don't add nonces to front-end forms, because if the user is logged out (which is true 99% of the time), they'll get cached and be invalid.

If we do add nonces, the generation and verification needs to be wrapped in `is_user_logged_in()` calls. I think it's fine to just leave them out, though. We could add a `// phpcs:ok missing_nonce` (or whatever the syntax is) to avoid PHPCS complaining, but should also include a note that the reason is because it'd get cached.

**Comment by iandunn** [comment]
Doh, what Corey said :)

**Comment by coreymckrill** [comment]
Also, what Ian said 😄 

> I think it's fine to just leave them out

File: public_html/wp-content/plugins/wcpt/views/applications/wordcamp/shortcode-application.php (hunk 1 above)
Comment (vedanshujain) [reply public_html/wp-content/plugins/wcpt/views/applications/wordcamp/shortcode-application.php]: decided to not include nonce in frontend app

**Pull request closed without merging**